modules/javafx.controls/src/main/java/javafx/scene/control/skin/TreeTableRowSkin.java

```diff
@@ -480,6 +483,7 @@ private static class StyleableProperties {
                 }
                 return null;
             }
+            // fall through
```

Comment:
There is no "fall through" here as we return from all conditions in previous case.